deploy/config/transfer-ownership.ts

```diff
@@ -0,0 +1,83 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { ConfigFileHelper } from "../file-helper/config-file-helper";
+import { getDeployer } from "../utils/deployer-helper";
+import { getProxyAdminFactory } from "@openzeppelin/hardhat-upgrades/dist/utils";
+import { AutomatedVaultManager__factory, Bank__factory } from "../../typechain/factories/src";
+import { PCSV3Executor01__factory } from "../../typechain/factories/src/executors";
+import { PancakeV3Worker__factory } from "../../typechain/factories/src/workers";
+import { PancakeV3VaultOracle__factory } from "../../typechain/factories/src/oracles";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const configFileHelper = new ConfigFileHelper();
+  const config = configFileHelper.getConfig();
+
+  /*
+  ░██╗░░░░░░░██╗░█████╗░██████╗░███╗░░██╗██╗███╗░░██╗░██████╗░
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+*/
+
+  const deployer = await getDeployer();
+
+  const opMultiSig = "0x18F59e8dDDef9e000863082a37fc56a2a5475D01";
```

Comment:
please put in .mainnet.json
```suggestion
  const opMultiSig = "0x18F59e8dDDef9e000863082a37fc56a2a5475D01";
```